integration/iid/iid_test.go

```diff
@@ -51,11 +51,12 @@ func TestMain(m *testing.M) {
 }
 
 func TestNonExisting(t *testing.T) {
-	err := client.DeleteInstanceID(context.Background(), "dnon-existY")
+	// legal instance IDs are /[cdef][A-Za-z0-9_-]{9}[AEIMQUYcgkosw048]/
```

Comment:
This doesn't look like it belong here. May be it will go away when merged with the latest dev?

Reply:
this explains fictive-ID0 , i'll change the comment